llvm/include/llvm/CodeGen/SelectionDAGNodes.h

```diff
@@ -475,6 +475,7 @@ struct SDNodeFlags {
   bool hasAllowReassociation() const { return Flags & AllowReassociation; }
   bool hasNoFPExcept() const { return Flags & NoFPExcept; }
   bool hasUnpredictable() const { return Flags & Unpredictable; }
+  bool hasFastMath() const { return Flags & FastMathFlags; }
```

Comment:
Should not add this. This does not depend on all the flags, and you should only depend on the exact set of flags required for a transform